usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -89,7 +102,6 @@ def handle(self, *args, **options):
             # See comment below about old date and time values cannot parsed without these
             "spark.sql.legacy.parquet.datetimeRebaseModeInWrite": "LEGACY",  # for dates at/before 1900
             "spark.sql.legacy.parquet.int96RebaseModeInWrite": "LEGACY",  # for timestamps at/before 1900
-            "spark.sql.jsonGenerator.ignoreNullFields": "false",  # keep nulls in our json
         }
```

Comment:
Looks like this line might have also been removed during a merge. Can you add it back?